mappings/net/minecraft/client/profiler/Recorder.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5962 net/minecraft/client/profiler/Recorder
```

Comment:
Should this be in `client.util.profiler` like the other classes?

Reply:
Whoops!